test/testcafe/spec/AuthenticatorYubikeyView_spec.js

```diff
@@ -65,13 +66,13 @@ test
     await t.expect(pageObject.getFormTitle()).eql('Set up YubiKey');
     
     // Do not fill out the form and submit
-    await pageObject.submit();
+    await pageObject.clickEnrollButton();
 
     pageObject.form.waitForErrorBox();
-    await t.expect(pageObject.form.getErrorBoxText()).eql('We found some errors. Please review the form and make corrections.');
+    await t.expect(pageObject.errorBoxTextExists()).eql(true);
```

Comment:
I think the previous assertion was more explicit/clear. If there's no way to grab the error box without knowing the text, that's the real problem.

Reply:
There should be some aria attrs that you can use. If not we have to add them (separate bug/PR)

Reply:
yeah i will see if I can find a better way

Reply:
Yeah it looks like since the error box is not inside the form, the getErrorBox function cant grab it.  And I dont see any aria attrs that i can use